docs/08_crosscutting_concepts/08_crosscutting_concepts.md

```diff
@@ -2,4 +2,212 @@
 title: Crosscutting concepts
 ---
 
-# Crosscutting concepts
\ No newline at end of file
+# Crosscutting concepts
+
+
+# Deployment view
+
+## Frontend
+
+### 8.1 Dependencies
+For the UI components:<br>
+https://ant.design/components/overview/
+
+For routing: <br>
+https://reactrouter.com/
+
+I18N Internationalization: <br>
+https://react.i18next.com/
+
+MSAL User authentication: <br>
+https://github.com/microsoft/Federal-App-Innovation-Community/tree/main/topics/modern-auth/React-MSAL-AAD
```

Comment:
This seems not to be the official repo: https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-react

Reply:
Replaced with your link